bigframes/core/global_session.py

```diff
@@ -26,8 +26,8 @@
 
 def close_session() -> None:
     """Start a fresh session the next time a function requires a session.
-
-    Closes the current session if it was already started.
+    Closes the current session if it was already started, deleting any
```

Comment:
This will be included as part of the first summary line. We want a new line above. (https://peps.python.org/pep-0257/#multi-line-docstrings)

```suggestion

    Closes the current session if it was already started, deleting any
```